packages/terra-date-picker/tests/wdio/date-picker-spec.js

```diff
@@ -328,4 +328,18 @@ describe('Date Picker', () => {
     Terra.should.matchScreenshot();
     Terra.should.beAccessible({ rules: ignoredA11y });
   });
+
+  describe('Key Limitations', () => {
+    before(() => browser.url('/#/raw/tests/terra-date-picker/date-picker/date-picker-key-limits'));
+
+    it('sets the date', () => {
+      browser.setValue('input[name="terra-date-date-input"]', '0');
+      browser.keys('a1.b2/;3');
+      // Ensures the mouse pointer doesn't appear in the screenshot
+      browser.click('h3');
+    });
```

Comment:
Since the DatePickerKeyLimits.test.jsx doesn't require to set any specific props, you could get rid of it and just use the [DatePickerDefault](https://github.com/cerner/terra-framework/blob/master/packages/terra-date-picker/src/terra-dev-site/test/date-picker/DatePickerDefault.test.jsx) test. You can do [this](https://github.com/cerner/terra-framework/blob/master/packages/terra-time-input/tests/wdio/time-input-spec.js#L114-L116) to make the cursor transparent and won't need the `h3`.

Reply:
Fixed https://github.com/cerner/terra-framework/pull/595/commits/9af826c35a73bb5fad6c3023e10fbe23063035a5